lib/natalie/compiler/instructions/push_rational_instruction.rb

```diff
@@ -0,0 +1,23 @@
+require_relative './base_instruction'
+
+module Natalie
+  class Compiler
+    class PushRationalInstruction < BaseInstruction
+      def to_s
+        'push_rational'
+      end
+
+      def generate(transform)
+        denominator = transform.pop
+        numerator = transform.pop
+        transform.exec_and_push(:rational, "Value(RationalObject::create(env, #{denominator}->to_int(env), #{numerator}->to_int(env)))")
```

Comment:
Not sure about the `to_int` here, but RationalObject only takes IntegerObject pointers so we kind of have to do that. However, I'm not sure if RationalObject should hold Values instead of IntegerObjects (for example subclasses of Integer?)

Reply:
This is fine for now. We can change it later if needed!